src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommandList.cs

```diff
@@ -0,0 +1,266 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using static Interop.VoiceControlManager;
+using static Interop.VoiceControlCommand;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This class represents a list of the voice commands.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceCommandList
```

Comment:
You need to implement IDisposable interface because it has IDisposable value in field

Reply:
Fixed.